packages/storage/README.md

```diff
@@ -53,6 +53,11 @@ localReadStream.pipe(remoteWriteStream);
 bucket.upload('/photos/zoo/zebra.jpg').then(function(data) {
   var file = data[0];
 });
+
+// It's also possible to integrate with third-party Promise libraries.
```

Comment:
I think we should actually leave this out for now, and wait for a more "global" place to document this that we can link to (either the main README or a new page in the Getting Started section of the docs site)